### PR TITLE
package chunk

### PR DESCRIPTION
**Pull request by elliott5**
Adds a simple Named Entity Extraction function.
See test for example use.

### CODE REVIEW

File: chunk/chunk.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			padding = "__"
+		case 3:
+			padding = "_"
+		case 4: // no padding required
+		default:
+			tag = tag[:4] // longer than 4 ... truncate!
+		}
+		tagQuads += tag + padding
+	}
+	//fmt.Println(tagQuads)
```
Comment (jdkato) [inline chunk/chunk.go]: Did you mean to leave this here?

**Comment by jdkato** [comment]
It looks good!

How would you like me to record this contribution? Should I add you to the LICENSE, create a of list contributors, or something else?

File: chunk/chunk.go (hunk 1 above)
Comment (elliott5) [inline chunk/chunk.go]: Ooops!
Nope, line 32 is redundant.

**Comment by elliott5** [comment]
I would like you to record my contribution. I leave it to you how, but I should think a contributors list would be easiest if others contribute too.

**Comment by jdkato** [comment]
Thanks!

**Comment by jdkato** [comment]
I've added an [`AUTHORS.md`](https://github.com/jdkato/prose/blob/master/AUTHORS.md) file, which seems to be a common way to list contributors. Feel free to suggest edits.

**Pull request merged**